hyper-rustls 0.27.9 doc build failed

https://docs.rs/crate/hyper-rustls/latest/builds/3138972

```
[INFO] [stderr] warning: target filter specified, but no targets matched; this is a no-op
[INFO] [stderr]  Documenting hyper-rustls v0.27.9 (/opt/rustwide/workdir)
[INFO] [stderr] error[E0557]: feature has been removed
[INFO] [stderr]   --> src/lib.rs:36:38
[INFO] [stderr]    |
[INFO] [stderr] 36 | #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
[INFO] [stderr]    |                                      ^^^^^^^^^^^^ feature has been removed
[INFO] [stderr]    |
[INFO] [stderr]    = note: removed in 1.92.0; see <https://github.com/rust-lang/rust/pull/138907> for more information
[INFO] [stderr]    = note: merged into `doc_cfg`
[INFO] [stderr] 
[INFO] [stderr] error: Compilation failed, aborting rustdoc
[INFO] [stderr] 
[INFO] [stderr] For more information about this error, try `rustc --explain E0557`.
[INFO] [stderr] error: could not document `hyper-rustls`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "hyper-rustls"
-version = "0.27.9"
+version = "0.27.10"
 edition = "2021"
 rust-version = "1.85"
 license = "Apache-2.0 OR ISC OR MIT"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,6 +53,16 @@ name = "server"
 path = "examples/server.rs"
 required-features = ["aws-lc-rs"]
 
+[lints.clippy]
+cloned_instead_of_copied = "warn"
+manual_let_else = "warn"
+needless_collect = "warn"
+needless_pass_by_ref_mut = "warn"
+or_fun_call = "warn"
+redundant_clone = "warn"
+upper_case_acronyms = "warn"
+use_self = "warn"
+
 [lints.rust]
 elided_lifetimes_in_paths = "warn"
 trivial_numeric_casts = "warn"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,6 +53,9 @@ name = "server"
 path = "examples/server.rs"
 required-features = ["aws-lc-rs"]
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ["cfg(hyper_rustls_docsrs)"] }
+
 [package.metadata.docs.rs]
 no-default-features = true
 features = [
@@ -65,4 +68,4 @@ features = [
     "tls12",
     "webpki-tokio",
 ]
-rustdoc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "hyper_rustls_docsrs"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,7 +54,14 @@ path = "examples/server.rs"
 required-features = ["aws-lc-rs"]
 
 [lints.rust]
+elided_lifetimes_in_paths = "warn"
+trivial_numeric_casts = "warn"
 unexpected_cfgs = { level = "warn", check-cfg = ["cfg(hyper_rustls_docsrs)"] }
+unnameable_types = "warn"
+unreachable_pub = "warn"
+unused_extern_crates = "warn"
+unused_import_braces = "warn"
+unused_qualifications = "warn"
 
 [package.metadata.docs.rs]
 no-default-features = true

--- a/src/config.rs
+++ b/src/config.rs
@@ -128,6 +128,7 @@ impl ConfigBuilderExt for ConfigBuilder<ClientConfig, WantsVerifier> {
 mod sealed {
     use super::*;
 
+    #[expect(unnameable_types)]
     pub trait Sealed {}
 
     impl Sealed for ConfigBuilder<ClientConfig, WantsVerifier> {}

--- a/src/connector.rs
+++ b/src/connector.rs
@@ -138,7 +138,7 @@ where
 }
 
 impl<T> fmt::Debug for HttpsConnector<T> {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("HttpsConnector")
             .field("force_https", &self.force_https)
             .finish()

--- a/src/connector/builder.rs
+++ b/src/connector/builder.rs
@@ -229,7 +229,7 @@ impl WantsProtocols1 {
         HttpsConnector {
             force_https: self.https_only,
             http: conn,
-            tls_config: std::sync::Arc::new(self.tls_config),
+            tls_config: Arc::new(self.tls_config),
             server_name_resolver: self
                 .server_name_resolver
                 .unwrap_or_else(|| Arc::new(DefaultServerNameResolver::default())),

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -32,7 +32,7 @@
 //! # fn main() {}
 //! ```
 
-#![warn(missing_docs, clippy::use_self)]
+#![warn(missing_docs)]
 #![cfg_attr(hyper_rustls_docsrs, feature(doc_cfg))]
 
 mod config;

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -32,7 +32,7 @@
 //! # fn main() {}
 //! ```
 
-#![warn(missing_docs, unreachable_pub, clippy::use_self)]
+#![warn(missing_docs, clippy::use_self)]
 #![cfg_attr(hyper_rustls_docsrs, feature(doc_cfg))]
 
 mod config;

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -33,7 +33,7 @@
 //! ```
 
 #![warn(missing_docs, unreachable_pub, clippy::use_self)]
-#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 
 mod config;
 mod connector;

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -33,7 +33,7 @@
 //! ```
 
 #![warn(missing_docs, unreachable_pub, clippy::use_self)]
-#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(hyper_rustls_docsrs, feature(doc_cfg))]
 
 mod config;
 mod connector;

--- a/src/stream.rs
+++ b/src/stream.rs
@@ -36,7 +36,7 @@ impl<T: rt::Read + rt::Write + Connection + Unpin> Connection for MaybeHttpsStre
 }
 
 impl<T: fmt::Debug> fmt::Debug for MaybeHttpsStream<T> {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match *self {
             Self::Http(..) => f.pad("Http(..)"),
             Self::Https(..) => f.pad("Https(..)"),
@@ -60,7 +60,7 @@ impl<T: rt::Read + rt::Write + Unpin> rt::Read for MaybeHttpsStream<T> {
     #[inline]
     fn poll_read(
         self: Pin<&mut Self>,
-        cx: &mut Context,
+        cx: &mut Context<'_>,
         buf: rt::ReadBufCursor<'_>,
     ) -> Poll<Result<(), io::Error>> {
         match Pin::get_mut(self) {

